### ISSUE TITLE
Update joomla 3.7.5 to Joomla 3.8 and lost Administrator Backend

### ISSUE THREAD

**Issue by marcoartica** [joomla/joomla-cms#18028]
(Please sorry for my bad english. I am learning english)
### Steps to reproduce the issue
I have updated my  joomla 3.7.5 on a server on godaddy, today I join in the administror backend and saw the message "we have and update to 3.8" and I didn't backup my website today, and I click the button "update" and get an error.

### Expected result
I expected the update to 3.8 is succerfull

### Actual result
I get this error:

Fatal error: Call to undefined method JAdminCssMenu::addChild() in /home/content/88/8430388/html/administrator/modules/mod_sppagebuilder_admin_menu/tmpl/default_enabled.php on line 16

### System information (as much as possible)

Web hosting linux on godaddy with PHP 5.4 

I can change php to 5.6 only

### Additional comments

The webpage works( the front-end) , but a can't access to the backend (site/administrator )
it's shows me the inputbox for password and username then I click login and get this:

HTTP ERROR 500

I need help :(

**Comment by brianteeman** [comment]
You need to update pagebuikder

https://www.joomshaper.com/forums/joomla-3-8

**Comment by marcoartica** [comment]
Thanks brianteeman.... Solved!!!!
that link talks about "rename the folder mod_sppagebuilder_admin_menu"
I'm so happy!

**Comment by Agustin-Pantoja** [comment]
Having the same problem after updating to 3.8 and I do not have this module (pagebuilder). Cannot access the backend. I get this error:

Error: 0 Call to undefined method JAdminCssMenu::renderMenu()

Any other help?

**Comment by ghost** [comment]
@Agustin-Pantoja have you read other the Threads to similar Issues?

**Comment by wojsmol** [comment]
@Agustin-Pantoja Do you have ssh access to the site?

**Comment by Agustin-Pantoja** [comment]
yes, I have read some similar threads... reading threads for the last five days...
and no, I do not have ssh access to the site. ;-)

**Comment by ghost** [comment]
@Agustin-Pantoja thanks for Clarification.

**Comment by Agustin-Pantoja** [comment]
Thanks for listening ;-)

**Comment by HanVanHeerde** [comment]
Like Agustin-Pantoja I do not have this module (pagebuilder). I read many articles and searched the internet. If any posts are made at all about this error, they are made about Pagebuilder. I do not have PageBuilder installed. Yet I do have "Call to undefined method JAdminCssMenu::addChild()" in the back-end of Joomla 3.8.1. I also updated all extensions etc., before updating from Joomla 3.7.5 to Joomla 3.8.1. Any ideas anyone please? Two sites have this problem. 4 others have not.
In addition. When I rename administrator/modules/mod_menu to administrator/modules/mod_menuold the error goes away, but also my back-end menu. I do see my Controlepaneel (=Dashboard?).

**Comment by wojsmol** [comment]
@HanVanHeerde Please create topic on forum https://forum.joomla.org/viewforum.php?f=710

**Comment by atualizado** [comment]
Hi friends,
Having the same problem after updating 3.7.x to 3.8.1 and I do not have this module (pagebuilder). 
Cannot access the backend. I get this error:
Error: 0 Call to undefined method JAdminCssMenu::renderMenu()

I did not install SPageBuilder, but the error presented was the same and the solution was similar, in my case, I checked the PHP logs and indicated the error in the 'CCK Module of SEBLOD 3.x', so I had to rename the folder "mod_cck_menu" to "mod_cck_menu_anything" and everything worked again.

Anyone with the same error being presented, I recommend that they see the PHP log file, they will have clarifications on the module that is causing the problem.

This solved my case, I hope it helps anyone who needs it ...

thank you ;-)

**Comment by brianteeman** [comment]
Was seblod up to date? If it was it would be useful if you have them feedback

**Comment by atualizado** [comment]
Hi
Yes, all extensions were already updated by auto-update in 'extensions> manager> update', but when updating Joomla the administrator screen displayed only top menu.
The page title displayed the reported error: Error: 0 Call to undefined method JAdminCssMenu::renderMenu()

Maybe I'd better post on the seblod forum,
I'm sorry for posting here.

**Comment by brianteeman** [comment]
Then seblod did you no favours by not testing and ensuring their software was up to date and compatible, Joomla 3.8 had a very long release cycle with multiple release candidates that they could and should have tested with

**Comment by BastianFie** [comment]
I had the same issue (Error: 0 Call to undefined method JAdminCssMenu::renderMenu()) with a Joomla installation with no 3rd party applications. I deleted the folder "modules/mod_menu" and "administrator/modules/mod_menu" and uploaded a more recent version (in my case Joomla 3.8.2) via FTP and the login is working again now and the error message went away. After that I went to /administrator/index.php?option=com_joomlaupdate and reinstalled the joomla core files (as I still saw some errors in the server logfile). So I assume at least in my case this was caused by wrong update process so its not an issue in joomla itself.

**Comment by Agustin-Pantoja** [comment]
Finally a friend of mine found the problem. The key is to check where is the problem. It has to be related to the menu of the template. 
I tried to check the log on the server, but not a  clue there. At the end I did point number 5 of this website.
https://www.collectiveray.com/joomla/joomla-tutorials/joomla-blank-page.html
In my case I  found out that I tried to override the menu of the backend some time ago, cause it had too many items. 
I went to /public_html/administrator/templates/isis/html and there there was a folder named mod_menu. I just renamed it and everything worked out again I could update the website, no problem. 
To get rid of the elements in administrator menu/components, better follow this guide:
https://www.joomlashack.com/blog/tutorials/deleted-components-menu-items-remained/


Hope it helps to someone.

**Comment by inport** [comment]
Hi, Help me, please,  with the site http://www.edu-ungheni.md/administrator/ - HTTP ERROR 500

**Comment by brianteeman** [comment]
Please use the forum for support

**Comment by ghost** [comment]
@inport if needed: [forums](https://forum.joomla.org/).

**Comment by Agustin-Pantoja** [comment]
Hi again, 

I had another blank page in the administrator backend after updating a different website from the one I mentioned above from joomla to joomla 3.9.2. This time the error was: Error 0 Call to undefined method JAdminCssMenu::addchild

I just checked my own post here and again I renamed the folder mod_menu in administrator/templates/isis/html. Now it works again. 

I am starting to think that this is a bug, as I had the very same problem in 2 different websites made by two different developers with different templates and different modules installed.
